src/Modal/Modal.js

```diff
@@ -51,9 +52,14 @@ class Modal extends Component {
             ...rest
         } = this.props;
 
-        const modalClasses = classnames(
+        const backdropClasses = classnames(
             'fd-overlay',
             'fd-overlay--modal',
+            backdropClassName
+        );
+
+        const modalClasses = classnames(
+            'modal-demo-bg',
```

Comment:
I know this class was here before, but should it be?  This seems like something used as a selector for unit tests and there is an entry in `customstyles.css` from `fundamental-styles`.  It's fine to use some custom styles for a playground or such, but it shouldn't be part of the base component's classes.

Reply:
Good catch! I'll remove it

Reply:
@greg-a-smith turns out that entire div was just for the styles docs site. I'm removing it here and going to make a pr there to make that more obvious.